chapter_appendix_math/statistics_vn.md

```diff
@@ -480,15 +485,24 @@ While beyond the scope of this brief appendix to derive in detail, as an example
 However, if we think our sample population true mean is close to $0.01$, then we'd need a sample size of nearly $80000$ to detect the difference.
 -->
 
-*dịch đoạn phía trên*
+Một trong những ứng dụng phổ biến nhất của năng lực thống kê là để xác định số lượng mẫu cần thiết.
+Xác suất bạn bác bỏ giả thuyết gốc khi nó là sai phụ thuộc vào mức độ sai của nó (hay còn gọi là *hệ số ảnh hưởng*) và số lượng mẫu bạn có.
+Và như bạn đã có thể đoán trước, hệ số ảnh hưởng nhỏ sẽ cần một số lượng mẫu rất lớn để có thể bị phát hiện với xác suất cao.
+Việc đi sâu vào chi tiết nằm ngoài phạm vi của phần phụ lục ngắn gọn này, nhưng đây là một ví dụ.
+Giả sử ta muốn bác bỏ một giả thuyết gốc rằng mẫu của ta được lấy từ một phân phối Gauss với kỳ vọng là 0, phương sai là 1, và ta tin rằng giá trị trung bình của tập mẫu gần với 1.
```

Comment:
```suggestion
Giả sử ta muốn bác bỏ một giả thuyết gốc rằng mẫu của ta được lấy từ một phân phối Gauss với kỳ vọng là không, phương sai là một, và ta tin rằng giá trị trung bình của tập mẫu gần với một.
```

Reply:
sao lại phải viết thế này nhỉ?

Reply:
Có một nguyên tắc là các số nhỏ dưới 10 đều nên viết thành chữ, bạn chỉ nên viết số khi thoả nguyên tắc trên hoặc nội dung này cần được mô tả dưới dạng chữ số. Trong trường hợp này thì không.

Reply:
Hồi xưa toàn bị trừ điểm môn văn vì lỗi này :))